project/ws/app/src/lib/routes/profile-v2/components/designation-request/designation-request.component.ts

```diff
@@ -58,16 +60,20 @@ export class DesignationRequestComponent implements OnDestroy {
     this.requestService.createPosition(postData)
     .pipe(takeUntil(this.destroySubject$))
     .subscribe((_res: any) => {
-      this.matSnackBar.open('Your designation request has been submitted successfully!')
+      this.matSnackBar.open(this.handleTranslateTo('designationRequestSent'))
       this.handleCloseModal()
     },         (error: HttpErrorResponse) => {
       if (!error.ok) {
-        this.matSnackBar.open('Unable to request given designation, please try again later')
+        this.matSnackBar.open(this.handleTranslateTo('designationRequestFailed'))
         this.handleCloseModal()
       }
     })
   }
 
+  handleTranslateTo(menuName: string): string {
```

Comment:
This function shold be in a service file, as we are using same function in multiple components